com.github.Nheko.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
Please don't sort this alphabetically, that isn't useful. See the layout of other applications.